democracy/models/gdpr_data_serialization_mixin.py

```diff
@@ -0,0 +1,13 @@
+class FileFieldUrlSerializerMixin:
+    field_to_use_as_url_field = None  # Override in subclass.
+
+    @property
+    def url(self):
+        from kerrokantasi.gdpr import _thread_locals
+
+        field = getattr(self, self.field_to_use_as_url_field, None)
+
+        if request := getattr(_thread_locals, "request", None):
```

Comment:
I think it would be better if we could hide the implementation details and maybe add a utility function like `get_current_request()` for getting the request.

```
def get_current_request():
    return getattr(_thread_locals, "request", None)
```

Reply:
surely 👍

Reply:
it is done.